app/controllers/errors_controller.rb

```diff
@@ -6,6 +6,11 @@ class ErrorsController < ApplicationController
   end
 
   def auth_error
-    @page_subtitle = "Auth Error"
+    @page_subtitle = t(".subtitle")
+  end
+
+  def timeout_error
+    @page_subtitle = t(".subtitle")
+    render "timeout_error", status: 504, formats: :html
```

Comment:
The format defaults to html, so it can be left out:
```suggestion
    render "timeout_error", status: :gateway_timeout
```